src/components/ui/header.tsx

```
@@ -0,0 +1,34 @@
+    <nav className="flex items-center">
+      <Image
+        src="/assets/merch-track_4.png"
+        width={83}
+        height={83}
+        alt="Logo"
+      />
+      <div className="text-4xl font-medium">
+        MerchTrack
+      </div>
+      <div className="m-auto space-x-20">
+        <a className="text-lg">About</a>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)